docs/user-documentation/moderne-platform/how-to-guides/reporting.md

```diff
@@ -0,0 +1,61 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Reporting
+
+## Overview
+
+Moderne offers some reports:
```

Comment:
Do we offer more than one report for general users? Are we going to be adding more? It feels a bit strange to me to say we offer some reports and then just list one. 

Also - why would a user want to run this report? This seems like an administrative thing but maybe I'm misunderstanding something.

Reply:
I don't think we have any plans to add more reports, but we could. I wanted to make the document so that it would be easy to add a new report.

I split it out of the admin page because it doesn't require admin permissions. I suspect the reasons to run it as a none admin or an admin are very similar